### PR TITLE
[bugfix] syntax error and strange follow-up error

### PR DESCRIPTION
**Pull request by haslersn**
`install.yml` had a syntax error where

```
when: "{{ item.name }} is not defined"
```

was written instead of:

```
when: item.name is not defined
```

This led to the `postgresql.service` not starting on `@mattermost01`. Apparently
the path to `pg_ctl` wasn't specified correctly. (I don't know why the service
was working before.)

The fix for that follow-up issue involved accessing the postgresql version
(`pg_ver.stdout`) because the correct path is version-dependent.

### CODE REVIEW

File: templates/postgresql.service.j2 [hunk 1]
```diff
@@ -5,9 +5,9 @@ After=network.target
 
 [Service]
 Type=forking
-ExecStart={{ postgres_pg_ctl }} -s -D {{ postgres_data_directory }} start -w -t 120
```
Comment (dasJ) [inline templates/postgresql.service.j2:8]: Please get rid of the old variable everywhere, e.g. in README.md and tasks/install.yml

**Review by dasJ — DISMISSED** [review]

File: templates/postgresql.service.j2 (hunk 1 above)
Comment (haslersn) [reply templates/postgresql.service.j2:8]: I added a commit (24954bf7ee35da3c7060b34f58cc92c574afd326) to the source branch which should do it.

File: tasks/install.yml [hunk 2]
```diff
@@ -76,11 +76,8 @@
...
-  when: "{{ item.name }} is not defined"
+  when: item.name is not defined
   with_items:
     - name: postgres_initdb
       value: initdb
-      default: postgres_default_initdb
-    - name: postgres_pg_ctl
-      value: pg_ctl
-      default: postgres_default_pg_ctl
+      default: postgres_default_initdb
```
Comment (SlothOfAnarchy) [inline tasks/install.yml]: Please add the missing newline

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: tasks/install.yml (hunk 2 above)
Comment (haslersn) [reply tasks/install.yml]: Done.

**Comment by dasJ** [comment]
@SlothOfAnarchy Review and merge please

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**